client/src/core/client/stream/App/TabBar.css

```diff
@@ -5,6 +5,13 @@
   padding-right: var(--spacing-3);
 }
 
+.condensedTab {
+  padding-top: var(--spacing-2);
```

Comment:
it can, I honestly pulled it from the `smallTab` that does the same and already existed. Just following the prior styles. I personally don't mind explicit styles for each direction (did it a lot with XAML and WPF apps)